### DISCUSSION TITLE
Vercel Deployments having issues with Next-Auth

### DISCUSSION THREAD

**Discussion by sbmsr** [vercel/community#1493] [Help] [answered]
### Summary

I've been using next-auth to let users login with github. Earlier today I noticed my users were no longer able to login after making a deploy sometime around 4PM UTC.

I posted an issue [here](https://github.com/nextauthjs/next-auth/issues/6685#issuecomment-1426559724), and have started to see a flood of people say they're also seeing the same behavior. [Someone even mentioned that migrating to Netlify has fixed the issue](https://github.com/nextauthjs/next-auth/issues/6685#issuecomment-1426535665). (Update: [I have verified that next-auth is working on Netlify](https://github.com/nextauthjs/next-auth/issues/6685#issuecomment-1426580104))

Is it possible for someone to take a look? This issue seems to only happen once someone redeploys to production. I worry as more people inevitably do so, the problem will get worse.

### Additional information

Here is a [public repo](https://github.com/sbmsr/github-issue-example) and a [live app](https://github-issue-example.vercel.app/) showing the issue.

### Example

https://github.com/sbmsr/github-issue-example

**Comment by mintavenger** [comment]
Instead of redeploying you should rollback. 

![image](https://user-images.githubusercontent.com/113479684/218233279-1e2343de-8932-4b55-bc58-94b1da906677.png)

This avoids rebuilding with whatever issue with callbacks is happening in the next auth js code.

We rolled back to an older version (switches the URL) instead of redeploying (rebuilds the code with the problematic auth issue).

**Reply by rapz** [reply]
I tryed the instant rollback to yesterdays deploy and got nothing

**Reply by sbmsr** [reply]
That's working for now, but will be a temp fix ofcourse. Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

We reverted a changes that we believe should fix the problem. Please redeploy your app, and let us know If you are still facing this issue: https://vercel.com/help
